manoseimas/mps_v2/static/components/row_components.jsx

```diff
@@ -129,7 +129,9 @@ var LobbyistRow = React.createClass({
         </div>
         <div className="two wide column">
           <div className="ui project statistic">
-            <div className="value">{lobbyist.law_project_count}</div>
+            <div className="value">
+              <a href={lobbyist.url}>{lobbyist.law_project_count}</a>
+            </div>
             <div className="label">
```

Comment:
It would be better if the link coloring of the numbers was consistent with the link coloring on the title. We should keep this coloring:

![image](https://cloud.githubusercontent.com/assets/4104268/10687262/7d1f596c-7974-11e5-94e9-aac33955c970.png)

as opposed to this coloring:

![image](https://cloud.githubusercontent.com/assets/4104268/10687270/88790ab0-7974-11e5-98e5-2fc3fd151c73.png)